tests/dp_engine_test.py

```diff
@@ -755,11 +755,9 @@ def test_run_e2e_local(self):
 
         self.assertEqual(5, len(list(output)))
 
-    @unittest.skipIf(
-        sys.platform == "win32" or
-        (sys.version_info.minor <= 7 and sys.version_info.major == 3),
-        "There are some problems with PySpark setup on older python and Windows"
-    )
+    @unittest.skip("There are some problems with serialization in this test. "
```

Comment:
I wonder if this is related to me not being able to extract the metrics (e.g., count, sum) in private_spark_test.py by treating the output of `dp_engine.aggregate()` as a named tuple. I was only able to reference them as a regular tuple (e.g., by calling `v[0]`).

I imagine we do want to use namedtuples in spark as well, so maybe something to look into?

Reply:
Yeah, it might be related. Let's do in the next PRs